backend/api/Controllers/RobotController.cs

```diff
@@ -709,16 +709,27 @@ public async Task<ActionResult> ResumeMission([FromRoute] string robotId)
 
         mission.UpdateWithIsarInfo(isarMission);
         mission.Status = MissionStatus.Ongoing;
-
         await _missionService.Create(mission);
 
         robot.Status = RobotStatus.Busy;
         robot.CurrentMissionId = mission.Id;
+
+        var deck = await _assetDeckService.ReadById(scheduleLocalizationMissionQuery.DeckId);
+
+        if (deck == null)
+        {
+            _logger.LogWarning("Could not find deck with id={id}", scheduleLocalizationMissionQuery.DeckId);
+            return NotFound("Deck not found");
+        }
+
+        deck.DeckName = scheduleLocalizationMissionQuery.DeckId;
+        deck.AssetCode = robot.CurrentAsset;
```

Comment:
Opposite. We don't want to  make changes to the assetDeck

Reply:
Would make more sense to set
robot.CurrentAsset = deck.AssetCode

Reply:
Done :)